sci-libs/libmed/libmed-4.0.0-r1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# EAPI=7 uses ninja generator by default but it's incompatible with USE=fortran
+# https://github.com/Kitware/ninja/tree/features-for-fortran#readme
+CMAKE_MAKEFILE_GENERATOR=emake
+
+FORTRAN_NEEDED=fortran
+# NOTE:The build for multiple python versions should be possible but complecated for the build system
+PYTHON_COMPAT=( python2_7 python3_{5,6} )
```

Comment:
what about `python3_7` there?

Reply:
Haven't tested python3_7 yet, as it's not yet stable. I will test it in an unstable chroot and see if it works.

Reply:
I can do it as well.

Reply:
I can do it, no worries. Already have an unstable chroot set up for such cases.
I compiles and runs the test suite and also loads when imported into python, so my guess is, it's working.